roles/chatbot/templates/chatbot.deployment.yaml.j2

```diff
@@ -99,20 +99,8 @@ spec:
       - name: ansible-chatbot
         image: {{ _chatbot_image }}
         imagePullPolicy: '{{ image_pull_policy }}'
-        command:
-          - /bin/sh
-          - -c
-          - |
-            if [ -z "$PROVIDER_VECTOR_DB_ID" ]; then
-              if [ -f "/rag/llama_stack_vector_db/provider_vector_db_id.ind" ]; then
-                export PROVIDER_VECTOR_DB_ID=$(cat /rag/llama_stack_vector_db/provider_vector_db_id.ind)
-                echo "Loaded PROVIDER_VECTOR_DB_ID: $PROVIDER_VECTOR_DB_ID"
-              else
-                echo "ERROR: PROVIDER_VECTOR_DB_ID is not set and file /rag/llama_stack_vector_db/provider_vector_db_id.ind was not found"
-                exit 1
-              fi
-            fi
-            exec /.llama/entrypoint.sh /usr/bin/python3.12
+        args:
+          [ 'sh', '-c', 'export $(cat $LLAMA_STACK_CONFIG_DIR/distributions/ansible-chatbot/provider_rag.env | xargs) && /.llama/entrypoint.sh /app-root/.venv/bin/python3.12' ]
```

Comment:
this might not be needed as it will set in the container image

Reply:
yes but only for downstream, but not upstream